.circleci/config.yml

```diff
@@ -97,8 +101,25 @@ step_appraisal_install: &step_appraisal_install
   run:
     name: Install Appraisal gems
     command: |
-      bundle exec appraisal clean # Ensure we fetch the latest version of dependencies
       bundle exec appraisal install
+step_appraisal_update: &step_appraisal_update
+  run:
+    name: Update Appraisal gems
+    command: | # Remove all generated gemfiles and lockfiles, resolve, and install dependencies again
+      bundle exec appraisal update
+ensure_lockfile_committed: &ensure_lockfile_committed
+  run:
+    name: Ensure Gem lock files are commited
+    command: |
+      CHANGED_FILES=$(git status --porcelain)
+      if [[ $CHANGED_FILES ]]
+      then
+        >&2 echo "Gem lock files were modified or new lock files were created during"
+        >&2 echo "'bundle install' and 'bundle appraisal install' installation in CI."
+        >&2 echo "You need to check in those changes in your branch."
+        >&2 echo "Affected files:"
+        >&2 echo $CHANGED_FILES
+      fi
```

Comment:
Will this just log a warning, or will it actually break the build? (my CI-foo fails me on this lol)

Reply:
Ha, nice point, I wanted it to break the build. I guess I forgot to raise the alarm while I was occupied writing the error message.